evm-contracts/app.config.json

```diff
@@ -4,6 +4,10 @@
   "contractAbstractionDir": ".",
   "useDockerisedSolc": false,
   "compilerSettings": {
+    "optimizer": {
```

Comment:
What does this do?

Reply:
This is the core of the change. These options weren't getting passed along, but we save a lot of space in the size of the compiled contract by with this configuration. (~40% savings based on the contract I was testing with.)